Apps/Sandcastle/gallery/Imagery Layers Split.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Use the split property to only show layers on one side of a slider.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Tutorials, Showcases">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+
+<style>
+    @import url(../templates/bucket.css);
+
+    #slider {
+        position: absolute;
+        left: 50%;
+        top: 0px;
+        background-color: #D3D3D3;
+        width: 2px;
+        height: 100%;
+        z-index: 9999;
+    }
+
+    #slider:hover {
+        cursor: ew-resize;
+    }
+
+</style>
+
+<div id="cesiumContainer" class="fullSize">
+    <div id="slider"></div>
+</div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+
+
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    imageryProvider : new Cesium.ArcGisMapServerImageryProvider({
+        url : 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Street_Map/MapServer'
+    }),
+    baseLayerPicker : false
+});
+
+var layers = viewer.imageryLayers;
+var blackMarble = layers.addImageryProvider(Cesium.createTileMapServiceImageryProvider({
+    url : 'https://cesiumjs.org/blackmarble',
+    credit : 'Black Marble imagery courtesy NASA Earth Observatory',
+    flipXY : true // Only old gdal2tile.py generated tilesets need this flag.
+}));
+blackMarble.split = -1.0; // Only show to the left of the slider.
+
+// Sync the position of the slider with the split position
+var slider = document.getElementById('slider');
+viewer.scene.frameState.splitPosition = (slider.offsetLeft) / slider.parentElement.offsetWidth;
```

Comment:
`frameState` is not part of the public API.  Can this, instead, be a property directly on `Scene` that is then used to set the property on `frameState` like other shadowed properties?